src/types/Advertisements.ts

```diff
@@ -6,7 +6,7 @@ import { Role } from "./Role";
 
 export interface Advertisement  {
     id: number;
-    name: String;
+    name: string;
```

Comment:
This solved a bug, where typescript of material ui, can't use `String` elements, but expects `string`

Reply:
Nice